basics/answers.md

```diff
@@ -101,15 +101,7 @@ isn't exactly like mine but it works just fine it's ok, and you can
     really long and it's hard to see what it does.
 
     The solution is string formatting. At the time of writing this, I
```

Comment:
My recommendation is probably not going to change any time soon now that we have f-strings

```suggestion
    The solution is string formatting. I
```